src/plot_api/plot_api.js

```diff
@@ -2586,7 +2586,7 @@ function makePlotFramework(gd) {
         .attr('id', 'defs-' + fullLayout._uid);
 
     fullLayout._topdefs = fullLayout._toppaper.append('defs')
-        .attr('id', 'defs-' + fullLayout._uid);
+        .attr('id', 'topdefs-' + fullLayout._uid);
```

Comment:
FYI @mdtusz I changed my mind.

We should make sure that our ids are unique on the page at all times.